### ISSUE TITLE
Twintris by Twinsen (POM '98) (PD) causes core error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#132]
```
ROM: Twintris by Twinsen (POM '98)
MD5: A74738D03ACA4E51690408D203E86D49

Happens with all 3 video plugins. Give error:

unknown task:
        type:0
        sum:214bf
        PC:8e0567c


```

Original issue reported on code.google.com by `sknau...@wesleyan.edu` on 12 Jul 2008 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
I'm pretty sure this is related to the rsp-hle plugin
```

Original comment by `jezte...@gmail.com` on 23 Dec 2008 at 6:18

**Comment by GoogleCodeExporter** [comment]
```
Just as update: Works with Z64gl LLE with Z64 RSP-LLE
```

Original comment by `sven@narfation.org` on 8 Oct 2009 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
This error was caused by RSP HLE plugin not forwarding GFX ucode to video 
plugin.
This has been fixed by commit 1e306f0.

```

Original comment by `bobby.sm...@gmail.com` on 31 Mar 2013 at 10:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `s...@narfation.org` on 18 Dec 2013 at 10:52
- Changed state: **Fixed**
